reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -501,8 +502,21 @@ class StaticHtmlReporter : Reporter() {
                     dl {
                         dd {
                             row.detectedLicenses.forEach { (finding, excludes) ->
+                                val firstFinding = finding.locations.first()
```

Comment:
A slight improvement could be to pick the locations which has the longest line length in order to prevent picking the one liner findings which often are just references or false detections. Just an idea, up to you to decide.

Reply:
Adding a single license finding for a detected license will like trick ORT users into believing that if the one finding we pick is invalid all other findings are invalid as well. My idea fro the StaticHtml reporter would be to show one finding for each cluster of similar findings plus change the UI to display these findings with CSS on mouse hover. We planning to do a refactor of the Reporter tool next month - wondering if it makes sense to make this change now

Reply:
> Adding a single license finding for a detected license will like trick ORT users into believing that if the one finding we pick is invalid all other findings are invalid as well.

Honestly, I don't think the majority of ORT user would be that naive.

> My idea fro the StaticHtml reporter would be to show one finding for each cluster of similar findings plus change the UI to display these findings with CSS on mouse hover.

Basically, I believe such fancy stuff should be left as an exercise for the Web App reporter. Of course I don't rule this out, but I believe this could also be an incremental improvement at a later time.

> We planning to do a refactor of the Reporter tool next month - wondering if it makes sense to make this change now

Actually, I think it's good to have this feature in before the refactoring to document what kind of features we need, and what needs to be considered as part of the refactoring.

Reply:
> > Adding a single license finding for a detected license will like trick ORT users into believing that if the one finding we pick is invalid all other findings are invalid as well.
> 
> Honestly, I don't think the majority of ORT user would be that naive.

I think changing the text from "link to the location" to "link to the first location" or "link to the first finding" would solve this problem.

Reply:
Turns out that I now need to changes in this PR, namely the `VcsHost` refactoring, in another upcoming PR. Would you be fine merging this and me improving the test later?

Reply:
Nevermind @mnonnenmacher, I've meanwhile updated the PR to include the requested test case.